src/chainparams.cpp

```diff
@@ -193,6 +193,14 @@ class CMainParams : public CChainParams {
                             //   total number of tx / (checkpoint block height / (24 * 24))
         };
 
+        // Hardcoded fallback value for the Sprout shielded value pool balance
+        // for nodes that have not reindexed since the introduction of monitoring
+        // in #2795.
+        nSproutValuePoolCheckpointHeight = 520633;
+        nSproutValuePoolCheckpointBalance = 22145062442933;
+        fZIP209Enabled = true;
+        hashSproutValuePoolCheckpointBlock = uint256S("0000000000c7b46b6bc04b4cbf87d8bb08722aebd51232619b214f7273f8460e");
```

Comment:
Why do we not need similar for Sapling?

Reply:
By the time Sapling activated, the non-deprecated releases of Zcashd at that time were tracking and recording value deltas in the block delta.  For the lifetime of Sprout, this wasn't the case.  See point 1 here: https://github.com/zcash/zcash/pull/3885